exonum-java-binding-time-service/src/test/java/com/exonum/binding/time/UtcZonedDateTimeSerializerTest.java

```diff
@@ -15,7 +15,11 @@
  *
  */
 
-package com.exonum.binding.time.serialization;
+package com.exonum.binding.time;
+
+import static org.hamcrest.CoreMatchers.equalTo;
```

Comment:
You can install https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml to have IDEA take care of that.

Reply:
Yes, thanks, I needed to reimport that.